test/main.js

```diff
@@ -36,7 +36,7 @@ const random = () => Math.random().toString(36).substring(7)
 
 const chromeCapabilities = Capabilities.chrome()
 const chromeOptions = {
-  'args': ['--use-fake-device-for-media-stream','--use-fake-ui-for-media-stream', `--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m`]
+  'args': ['--use-fake-device-for-media-stream','--use-fake-ui-for-media-stream', '--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m', '--ignore-certificate-errors']
```

Comment:
`'--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m'` won't work after changing the back-tick ` to single quote '. the back-tick is needed to allow for the embedded parameter.

Reply:
Thanks!